dev-python/pygame/python_pygame-1.9.4.recipe

```diff
@@ -47,8 +49,9 @@ BUILD_REQUIRES="
 	"
 
```

Comment:
There shouldn't be a newline here.

Reply:
Huh? Yes there should, this is between PREREQUIRES and BUILD().